packages/angular/projects/ui-angular/src/lib/primitives/button/button.component.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { ButtonComponent } from './button.component';
+import { TestBed, ComponentFixture } from '@angular/core/testing';
+
+describe('ButtonComponent', () => {
+  let fixture: ComponentFixture<ButtonComponent>;
+  let component: ButtonComponent;
+
+  beforeEach(async () => {
```

Comment:
Since there is just a single test in this file, we should be okay to remove the `beforeEach`

Reply:
OK I'll remove it

Reply:
Let's keep it for now. Jest gets confused without surrounding the Testbed in its own async function. I was getting `    A "describe" callback must not return a value.` after I removed it. 

And it's a good example of how to write tests in Angular.